app/models/saved_claim/burial.rb

```diff
@@ -3,18 +3,7 @@
 require 'pension_burial/processing_office'
 
 class SavedClaim::Burial < SavedClaim
-  FORM = '21P-530'
-
-  # attribute name is passed from the FE as a flag, maintaining camel case
-  attr_accessor :formV2 # rubocop:disable Naming/MethodName
-
-  after_initialize do
-    self.form_id = if Flipper.enabled?(:va_burial_v2)
-                     formV2 || form_id == '21P-530V2' ? '21P-530V2' : self.class::FORM.upcase
-                   else
-                     self.class::FORM.upcase
-                   end
-  end
+  FORM = '21P-530EZ'
```

Comment:
@wayne-weibel I was under the impression we had to run a script on production to update all the form_ids on SavedClaims to have the EZ instead of the V2.

Reply:
And we have the rake task for IPFs if needed

Reply:
Update: Although now im thinking about how the JSON will already be in there and we won't be able to just update the form_id. Open to suggestions on this one. Is this scenario when the BenefitsIntake retries or your ManualRemediation process?

Reply:
Yes that it what I am mainly thinking of, I am not sure if any 'shared' jobs or libraries are driven from that constant

Reply:
@wayne-weibel What script do you usually run to reprocess a claim? I'll double check